### PR TITLE
Support animated sticker validation

### PR DESCRIPTION
**Pull request by rob93c**
Stickerify is missing a way to understand if an animated sticker is already valid for Telegram: in that case it would answer its _error_ message.

This pull request aims to support for their validation, hence returning (in case of compliant files) the _file already valid_ answer.

Telegram's requirements are specified [here](https://core.telegram.org/stickers#creating-animations).

Out of them, these are the requirements enforced in the pull request:
- [x] The canvas size must be 512х512 pixels.
- [ ] Objects must not leave the canvas.
- [x] Animation length must not exceed 3 seconds.
- [ ] All animations must be looped.
- [x] Final file size must not exceed 64 KB after rendering in Bodymovin.
- [x] All animations must run at 60 Frames Per Second.

As part of this work, [Gson](https://github.com/google/gson) has been added as a dependency to parse JSON files.

### CODE REVIEW

File: src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java [hunk 1]
```diff
@@ -75,6 +93,71 @@ private static String detectMimeType(File file) {
...
+	 * @param file the file to check
+	 * @param mimeType the MIME type of the file
+	 * @return {@code true} if the file is compliant
+	 */
+	private static boolean isAnimatedStickerCompliant(File file, String mimeType) throws TelegramApiException {
+		if ("application/gzip".equals(mimeType)) {
+			String zipContent = "";
+
+			try (var gzipInputStream = new GZIPInputStream(new FileInputStream(file));
+				 var scanner = new Scanner(new InputStreamReader(gzipInputStream, StandardCharsets.UTF_8))) {
```
Comment (MartelliEnrico) [inline src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java:110]: JDK 9 introduces `InputStream#readAllBytes()` which can be used to create a new String using the constructor `String(byte[], Charset)`. This way we can avoid creating a scanner.

File: src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java [hunk 2]
```diff
@@ -75,6 +93,71 @@ private static String detectMimeType(File file) {
...
+			} catch (IOException e) {
+				LOGGER.atError().log("Unable to retrieve gzip content from file {}", file.getName());
+			}
+
+			try {
+				var sticker = GSON.fromJson(zipContent, AnimationDetails.class);
+
+				return isAnimationCompliant(sticker) && isFileSizeLowerThan(file, ANIMATION_FILE_SIZE);
+			} catch (JsonSyntaxException e) {
+				return false;
```
Comment (MartelliEnrico) [inline src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java:123]: This return is not strictly necessary.

**Review by MartelliEnrico — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java (hunk 1 above)
Comment (rob93c) [reply src/main/java/com/github/stickerifier/stickerify/media/MediaHelper.java]: Great suggestion, I just implemented it ✔️

**Review by MartelliEnrico — APPROVED** [review]
LGTM

**Pull request merged**